components/dash-core-components/tests/integration/graph/test_graph_varia.py

```diff
@@ -847,3 +859,77 @@ def graph_dims():
     assert graph_dims() == responsive_size
 
     assert dash_dcc.get_logs() == []
+
+
+def test_grva010_external_mathjax_prevents_lazy(dash_dcc):
+    app = Dash(
+        __name__,
+        eager_loading=False,
+        external_scripts=["https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-svg.js"],
+    )
+
+    app.layout = html.Div(id="div", children=[html.Button(id="btn")])
+
+    @app.callback(Output("div", "children"), [Input("btn", "n_clicks")])
+    def load_chart(n_clicks):
+        if n_clicks is None:
+            raise PreventUpdate
+
+        return dcc.Graph(mathjax=True, id="output", figure={"data": [{"y": [3, 1, 2]}]})
+
+    dash_dcc.start_server(
+        app,
+        debug=True,
+        use_reloader=False,
+        use_debugger=True,
+        dev_tools_hot_reload=False,
+    )
+
+    # Give time for the async dependency to be requested (if any)
+    time.sleep(2)
```

Comment:
```suggestion
    dash_dcc.wait_for_element("button#btn")
```
I hate to add explicit waits... can't we just wait for the content on the page to be complete?

Same below, we should have the graph invoke MathJax, then just test for the presence of a MathJax element in the graph.